src/bust.py

```diff
@@ -88,6 +92,22 @@ def stop(self) -> None:
         self.bust_stopped = True
         if self.play_song_task:
             self.play_song_task.cancel()
+        self.delete_temp_audio()
+
+    def delete_temp_audio(self) -> None:
+        if os.path.exists(self.temp_audio_file):
+            os.remove(self.temp_audio_file)
+
+    def seek_and_convert_to_opus(self, timestamp: int, local_filepath: str) -> None:
+        song_len = song_utils.get_song_length(local_filepath)
+        if(timestamp >= song_len):
+            timestamp = 0
+            print("Attempted to seek past length of song. Ignoring timestamp.")
+
+        ffmpeg_command = [
+            'ffmpeg', '-i', local_filepath, '-ss', str(timestamp), '-t', str(song_len - timestamp), '-c:a', 'libopus', '-b:a', '128k', '-y', self.temp_audio_file
```

Comment:
I haven't tried this, but I think the `-t` flag part isn't needed? I think it should go from the `-ss` time to the end automatically, though I have not tested this and can do so later. Setting an end time like this feels like it could pootentially chop some samples or so off the end in some cases due to imprecision